inspirehep/config.py

```diff
@@ -334,6 +338,74 @@
     "980__a": ["collections.primary"],
     "980__b": ["collections.secondary"],
     "542__l": ["information_relating_to_copyright_status.copyright_status"],
+    
+    "037__c": ["arxiv_eprints.categories"],
+    "246__a": ["titles.title"],
+    "595": ["hidden_notes"],
+    "650__a": ["subject_terms.term"],
+    "695__a": ["thesaurus_terms.keyword"],
+    "773__y": ["publication_info.year"],
+    "anyfield": ["external_system_numbers.value",
+                 "external_system_numbers.obsolete",
+                 "report_numbers.value", "arxiv_eprints.categories",
+                 "languages", "authors.full_name", "authors.alternative_name",
+                 "authors.affiliations.value",
+                 "title_variation", "breadcrumb_title", "titles.title",
+                 "thesis.degree_type", "thesis.university",
+                 "thesis.date", "thesis_supervisor.affiliation",
+                 "public_notes.value", "abstracts.value",
+                 "free_keywords.value", "accelerator_experiments.experiment",
+                 "thesaurus_terms.keyword", "collaboration",
+                 "publication_info.recid", "publication_info.page_artid",
+                 "publication_info.journal_issue",
+                 "publication_info.conf_acronym",
+                 "publication_info.journal_title",
+                 "publication_info.reportnumber",
+                 "publication_info.confpaper_info",
+                 "publication_info.journal_volume", "publication_info.cnum",
+                 "publication_info.pubinfo_freetext", "publication_info.year",
+                 "publication_info.isbn", "publication_info.note",
+                 "isbns.value", "imprints.place", "imprints.publisher",
+                 "imprints.date", "thesis_supervisor.full_name",
+                 "title_translation.title", "title_translation.subtitle",
+                 "preprint_date", "corporate_author",
+                 "titles.subtitle", "subject_terms.term", "dois.value"
+                 ],
+    "authorcount": ["authors.full_name"],
```

Comment:
All the `*count` fields, should actually be implemented with dedicated fields in ES, representing the number of instances of the corresponding field. We should implement this probably with Groovy. We can post-pone it.